src/com/google/common/css/compiler/gssfunctions/ColorUtil.java

```diff
@@ -43,6 +45,39 @@ public static Color hsbToColor(float[] inputHsb) {
     return Color.getHSBColor(inputHsb[H], inputHsb[S], inputHsb[B]);
   }
 
+  public static float[] hsbToHsl(float[] inputHsb) {
+    float hHsb = inputHsb[H];
+    float sHsb = inputHsb[S];
+    float bHsb = inputHsb[B];
+
+    float hHsl = hHsb;
+    float lHsl = bHsb * (2 - sHsb) / 2;
+    float sHsl = bHsb * sHsb / (1 - Math.abs(2 * lHsl - 1));
+
+    float[] hsl = new float[3];
+    hsl[H] = hHsl;
+    hsl[S] = sHsl;
+    hsl[L] = lHsl;
+
+    return hsl;
+  }
+
+  public static float[] toHsl(Color color) {
+    return hsbToHsl(toHsb(color));
+  }
+
+  public static Color hslToColor(float[] inputHsl) {
```

Comment:
I prefer read in caller code `hslToColor(hsl)` than `hslToColor(hslToHsb(hsl))` which can be an implementation detail.
But I can add a `hslToHsb` and change `hslToColor` to call `hslToHsb`.
So the code of `ColorUtil` will be more _symmetric_:

- `hsbToHsl` and `hslToHsb`
- `toHsb` and `toHsl`
- `hsbToColor` and `hslToColor`